aya/src/obj/btf/btf.rs

```diff
@@ -453,6 +477,13 @@ impl Btf {
                     // Start DataSec Fixups
                     let sec_name = self.string_at(ty.name_off)?;
                     let name = sec_name.to_string();
```

Comment:
to_string() isn't needed here

Reply:
It is as we need it to be `String` in `section_sizes.get` later on...
otherwise we have to use `section_sizes.get(&name.to_string())` and another `name.to_string()` in the error message that follows :thinking: